enterprise/models.py

```diff
@@ -65,6 +65,7 @@ class Meta(object):
 
     uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Enterprise Customer name."))
+    catalog = models.PositiveIntegerField(null=True, help_text=_("Course catalog for the Enterprise Customer."))
```

Comment:
Hmm. I'll take a look and see if we can query the course discovery API with some sort of alternate string. I don't _think_ it's supported though; if not, we'll just have to leave it as-is, especially since the administrator of the course discovery API can change the name of a catalog whenever they feel like it.